tox.ini

```diff
@@ -61,7 +61,15 @@ commands =
     sphinx-build {posargs:-E} -b html docs dist/docs
     sphinx-build -b linkcheck docs dist/docs
 
+[testenv:pdf]
+description = Build PDF documentation with Sphinx
+deps = sphinx
+commands =
+    sphinx-build -b latex docs/ docs/build/latex
+    make -C docs/build/latex all-pdf
```

Comment:
Let's put built artifacts into `dist/docs/latex` or `dist/docs/pdf`, i.e., under `dist/docs` instead of under the `docs` directory.